l10n_ar_sale/models/sale_order.py

```diff
@@ -73,6 +73,16 @@ def _create_invoices(self, grouped=False, final=False, date=None):
         # Intentamos Completar el dato tipo de documento si no seteado
         to_fix = invoices.filtered(lambda x: x.l10n_latam_use_documents and not x.l10n_latam_document_type_id)
         to_fix._compute_l10n_latam_available_document_types()
+        if self.is_module_installed("sale_subscription_ux"):
+            for invoice in invoices:
+                so = invoice.invoice_line_ids[0].sale_line_ids.order_id or False
+                if so and so.plan_id.bill_end_period:
```

Comment:
Runbot returns this error:
AttributeError: 'sale.subscription.plan' object has no attribute 'bill_end_period'. Did you mean: 'billing_period'?